src/datasets/meshes/mpi_faust.jl

```diff
@@ -0,0 +1,161 @@
+function __init__faust()
+    DEPNAME = "MPI-FAUST"
+    DOCS = "http://faust.is.tue.mpg.de/"
+
+    register(ManualDataDep(
+        DEPNAME,
+        """
+        Dataset: $DEPNAME.
+        Website: $DOCS
+        """,
+    ))
+end
+
+"""
+    FAUST(split=:train; dir=nothing)
+
+The MPI FAUST dataset(2014).
+
+FAUST contains 300 real, high-resolution human scans of 10 different subjects in 30 different poses, with automatically computed ground-truth correspondences.
+
+Each scan is a high-resolution, triangulated, non-watertight mesh acquired with a 3D multi-stereo system.
+
+FAUST is subdivided into a training and a test set. The training set includes 100 scans (10 per subject) with their corresponding ground-truth alignments. The test set includes 200 scans. The FAUST benchmark defines 100 preselected scan pairs, partitioned into two classes – 60 requiring intra-subject matching, 40 requiring inter-subject matching.
+
+The dataset required to be downloaded manually from the [website](http://faust.is.tue.mpg.de/) and extracted in the correct location. For information about where to place the dataset, refer to the example section.
+
+# Dataset Variables
+
+- `scans`: Vector of non-watertight scans in the form of `Mesh`.
```

Comment:
Not clear here what a `Mesh` (object) is.

Reply:
Added reference to Meshes.jl to the mesh doc page.